common/plugin.go

```diff
@@ -55,6 +57,7 @@ func NewPlugin(name, version string) (*Plugin, error) {
 func (plugin *Plugin) Initialize(config *PluginConfig) error {
 	plugin.ErrChan = config.ErrChan
 	plugin.Store = config.Store
+	plugin.Stateless = config.Stateless
```

Comment:
where `plugin.Stateless` getting used? Is this passed to `networkManager` structure?

Reply:
Yes, it will be used in the networkManager structure. But we will create a different PR for `network` package changes.
ref: https://github.com/Azure/azure-container-networking/pull/2142/files#diff-e8f6f6482a7db7a4cf3deff0b4d81e4e413868039fb70793702b04e95a0b0b38R128-R130